templates/portfolios/edit_form.html

```diff
@@ -22,6 +22,11 @@ <h2>Edit Portfolio</h2>
                     {% bootstrap_form form %}
                     <p><button type="submit" id="create-portfolio-button" class="btn btn-success">Update</button>
                         <a type="button" class="btn btn-default" href="/portfolios">Cancel</a>
+                        {# Show delete button if portfolio is empty and user can change portfolio #}
+                        {% if can_change_portfolio and portfolio.projects.all|length == 0 %}
+                            &nbsp;&nbsp;&nbsp;
```

Comment:
To reduce technical debt, we've begun avoiding the use of `&nbsp;` as a spacing mechanism. This can probably be moved into CSS.